ydb/core/tx/schemeshard/schemeshard__operation_alter_login.cpp

```diff
@@ -225,8 +225,10 @@ class TAlterLogin: public TSubOperationBase {
                     context.OnComplete.PublishToSchemeBoard(OperationId, parent->PathId);
                 }
             }
-            // TODO: publish only changed paths
-            context.OnComplete.PublishToSchemeBoard(OperationId, pathId);
+            NACLib::TACL effectiveACL(path->CachedEffectiveACL.GetForSelf());
+            if (effectiveACL.HasAccess(user)) {
```

Comment:
I think its worth explaining (by code comment) why this would work as intended. And what is the intention.